pom.xml

```diff
@@ -210,12 +210,11 @@
       <artifactId>guava</artifactId>
       <version>18.0</version>
     </dependency>
-    <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>1.2.16</version>
-      <scope>provided</scope>
-    </dependency>
+      <dependency>
```

Comment:
This change was an attempt to get IntelliJ to be able to run something. It was not intended as a permanent change.